src/app/components/Menu/components/MenuItem/index.tsx

```diff
@@ -6,7 +6,7 @@ import React, {
   useCallback,
 } from 'react';
 import classNames from 'classnames';
-import { Link } from 'react-router-dom';
+import { Link, useLocation } from 'react-router-dom';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
Since Icon component already merged into the dev branch i think make sense to replace FontAweasomeIcon with our Icon component

Reply:
Replaced, thanks for the suggestion.